src/message.ts

```diff
@@ -91,7 +91,7 @@ export class MessageComputer {
         }
 
         const version = packedMessage.version || DEFAULT_MESSAGE_VERSION;
-        const signer = packedMessage.signer ?? SDK_JS_SIGNER;
+        const signer = packedMessage.signer ?? "unknown";
```

Comment:
Can be a constant, near the constant `SDK_JS_SIGNER`. E.g. `UNKNOWN_SIGNER`.

Reply:
added a const called `UNKNOWN_SIGNER`.